asset-transfer-basic/chaincode-external-nano/connection.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
The prepackaged external builders have a richer semantics here, and can template the address etc. 

as an FYI - a lot of the external builder scripts are wrong with their handling of TLS.

Reply:
Yes, as a matter of fact this PR doesn't really provide for TLS support.

Reply:
yes - and that's perfectly fine for this example... TLS would just make it too complex